rbx_types/src/variant.rs

```diff
@@ -126,6 +126,7 @@ make_variant! {
     Vector3(Vector3),
     Vector3int16(Vector3int16),
     OptionalCFrame(Option<CFrame>),
+    QDir(std::path::PathBuf),
```

Comment:
Should we newtype this? I'm a little afraid of the format eventually growing a not-weird-Qt-type path. We can sort of follow the lead from `Content` here.